lib/vagrant-libvirt/action/set_name_of_domain.rb

```diff
@@ -51,7 +51,7 @@ def build_domain_name(env)
               # don't have any prefix, not even "_"
               ''
             else
-              config.default_prefix.to_s.dup.concat('_')
+              config.default_prefix.to_s.dup
```

Comment:
You'll probably need to update the tests in `spec/unit/action/set_name_of_domain_spec.rb` with this change

Reply:
Heh.  Yes, I just discovered this and pushed an update.

Thanks for the heads-up, and quickly at that!